sbt-plugin/src/sbt-test/sbt-akkaserverless/compile-only/test

```diff
@@ -149,5 +149,8 @@ $ exists target/scala-2.13/src_managed/main/com/example/replicated/set/domain/So
 $ exists src/test/scala/com/example/valueentity/domain/SomeValueEntitySpec.scala
 $ exists src/test/scala/com/example/valueentity/SomeValueEntityServiceIntegrationSpec.scala
 
+# Descriptor set must be included in classpath
+$ must-mirror target/scala-2.13/classes/protobuf/descriptor-sets/user-function.desc target/scala-2.13/resource_managed/main/protobuf/descriptor-sets/user-function.desc
```

Comment:
Without the fix it fails here and for some reason in that state the target file is a directory and not a file...